### PR TITLE
Minor CNI v1.1 api fixes

### PR DESCRIPTION
**Pull request by squeed**
A few small things I discovered I needed while implementing spec v1.1 for the plugins.

### CODE REVIEW

File: pkg/types/types.go [hunk 1]
```diff
@@ -68,6 +68,14 @@ type NetConf struct {
 
 	RawPrevResult map[string]interface{} `json:"prevResult,omitempty"`
 	PrevResult    Result                 `json:"-"`
+
+	// ValidAttachments is only supplied when executing a GC operation
+	ValidAttachments []GCAttachment `json:"cni.dev/valid-attachments"`
+}
+
+type GCAttachment struct {
```
Comment (henry118) [inline pkg/types/types.go:76]: Suggest including a doc here for the type

**Review by s1061123 — CHANGES_REQUESTED** [review]

**Comment by squeed** [comment]
@henry118 @s1061123 fixed, PTAL

**Review by henry118 — APPROVED** [review]
LGTM

**Review by MikeZappa87 — APPROVED** [review]

**Review by s1061123 — APPROVED** [review]
LGTM!

**Pull request merged**